dbt/models/iasworld/schema/iasworld.aasysjur.yml

```diff
@@ -0,0 +1,85 @@
+sources:
+  - name: iasworld
+    loaded_at_field: date_parse(wen, '%Y-%m-%d %H:%i:%s.0')
```

Comment:
One downside of this approach is that top-level keys such as `loaded_at_field` and `tags` need to be replicated across each file.

Reply:
**[Praise]** That doesn't seem to bad to me! If we have to choose between duplicating these fields and requiring that our teammates navigate 5k+ lines of code to find them, I think duplication is probably the better choice anyway.